### PR TITLE
[Instrumentation.Hangfire] Update CHANGELOG for 1.0.0 release

### PR DESCRIPTION
**Pull request by fred2u**
## Changes
Update CHANGELOG for 1.0.0 release

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## Unreleased
+## 1.0.0
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:3]: Please keep Unreleased as empty section.

```suggestion
## Unreleased

## 1.0.0
```

**Review by Kielek — COMMENTED** [review]
@open-telemetry/dotnet-contrib-maintainers, could you please check release request?

File: src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.0.0
```
Comment (cijothomas) [inline src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:5]: are you relying on any semantic conventions from otel? if yes, we'll need to hold off from 1.0.0 stable. If not, we need to document the attribute/tag names in the readme, before stable release.

Comment (utpilla) [reply src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:5]: @fred2u ^

Comment (fred2u) [reply src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:5]: @cijothomas No, I'm not

**Pull request merged**